src/ert/scheduler/openpbs_driver.py

```diff
@@ -263,7 +264,17 @@ async def kill(self, realizations: Iterable[int]) -> None:
             raise RuntimeError(process_message)
 
     async def poll(self) -> None:
+        last_successful_poll = time.time()
+        last_error_message = "unknown error"
+        has_warned_evaluator_of_polling_error = False
         while True:
+            if (
```

Comment:
Can this be moved to a helper function in the superclass to avoid code duplication?